interactive-player/Interactive Viewer/Views/StreamDetailInputScreen/StreamDetailInputScreen.swift

```diff
@@ -394,25 +415,31 @@ struct StreamDetailInputScreen: View {
         let disableAudio = false
         let videoQuality = VideoQuality.auto
         let saveLogs = false
+        let playoutDelayMin = showPlayoutDelay ? UInt(minPlayoutDelay) : nil
+        let playoutDelayMax = showPlayoutDelay ? UInt(maxPlayoutDelay) : nil
+        let maxBitrate: UInt = isShowingMaxBitrate ? UInt(maxBitrateString) ?? 0 : 0
 
         RecentStreamCell(streamDetail: SavedStreamDetail(
             accountID: accountID,
             streamName: streamName,
             subscribeAPI: productionSubscribeURL,
             videoJitterMinimumDelayInMs: jitterMinimumDelayMs,
-            minPlayoutDelay: nil,
-            maxPlayoutDelay: nil,
+            minPlayoutDelay: playoutDelayMin,
+            maxPlayoutDelay: playoutDelayMax,
```

Comment:
The rest of the debug options in this screen is only applied to the stream details inputted by the user, these three options should also be the same ?

Reply:
If you look at line 418 & 419, it will only show values if there have been values add by the use, otherwise it will be `nil`.
```suggestion
        let playoutDelayMin = showPlayoutDelay ? UInt(minPlayoutDelay) : nil
        let playoutDelayMax = showPlayoutDelay ? UInt(maxPlayoutDelay) : nil
```

Reply:
Yes I understand and it's a good thing in my opinion too. 

However the demo stream doesn't currently use the other debug options changed by the user, for example - url, jitter buffer, disable audio etc. These values are all set to use the defaults.

So my thought is - for the demo stream we should align all debug options to either use the `defaults(which is how it is now)` or the `values that the user input (which is what you are doing for these three options)` 

cc @RameshDPrasad can take a call on this one ?

Reply:
Passing nil is ok in that case SDK will not send anything to server and server will use its own default.
Can you confirm this @Yousif-CS

Reply:
I misunderstood the discussion here. For demo stream do not apply any debug config